commercetools-models/src/main/java/io/sphere/sdk/taxcategories/ExternalTaxRateDraftBuilder.java

```diff
@@ -16,6 +16,7 @@
     private String state;
     @Nullable
     private List<SubRate> subRates;
+    private Boolean includedInPrice;
```

Comment:
Shouldn't this also declared nullable or set to a default value? Documentation says defaults to false

Reply:
you're right with regards to the docs. But if the field is set to `null`, then the backend will set it to the default value `false`. And I don't think it makes sense to replicate this in our JVM SDK. Hope that makes sense to you too!